src/client/pages/OfferNew/Checkout/hooks.ts

```diff
@@ -45,3 +46,22 @@ export const useScrollLock = (
       window.removeEventListener('touchmove', listener)
     }
   }, [outerWrapperRef, visibilityState])
+
+export const useSsnError = (editQuoteResult: EditQuoteMutationResult) => {
+  const [ssnBackendError, setSsnBackendError] = useState('')
+  const editQuote = editQuoteResult.data?.editQuote
+
+  useEffect(() => {
+    if (
+      editQuote?.__typename === 'UnderwritingLimitsHit' &&
+      editQuote?.limits.length > 0
+    ) {
+      const { code } = editQuote.limits[editQuote.limits.length - 1]
```

Comment:
Since we get an array of limits for certain `UnderwritingLimitHits`, for reference a non matching ssn and birthdate returns this. If you have a better idea, shoot 😊  
<img width="682" alt="Screenshot 2020-12-07 at 15 20 17" src="https://user-images.githubusercontent.com/6661511/101361946-db971800-389f-11eb-9733-a61c100222e2.png">

Reply:
Hm, I guess if you would use this hook somewhere else, say in the modal for editing your details, you could potentially get more limits back and then we can't know that it's the last one that we want to set as  `ssnBackendError`. So that might be a concern - that this hook only works if you're not trying to edit more quote properties than the ssn. But on the other hand this file is in the `/Checkout` folder so maybe it's clear enough that the hooks here are not intended to be used anywhere else. 🤔

Reply:
I think it's fine as long as it's a local hook, in this specific folder. And since we can only get three different limits, or two really I think it covers the specific cases.

Reply:
> > > Yes, we should add validation for that for sure. 🙌 I just wanted to make this in batches 😊
> > 
> > 
> > @gustaveen Nice! So you don't think we should remove it (meaning the validation we do on the frontend with the `createSsnValidator`) and just let the backend do that kind of validation?
> 
> I'm not sure what I think 😄 Frontend validation is way faster so from a user perspective it's better. And I guess it's somewhat good to avoid mutations we know will fail. But it's a bit of a hustle to handle both frontend and backend error messages simultaneously 😅 So in that sense it's easier to handle all in the backend

Side note thought
If we are going to reuse some/all of this in the details modal, which currently only fires the mutations when clicking update, a diskussion regarding reuse of input fields and validation would be great and how we want to go about it.